packages/integration-matomo/src/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import {
+  getForegroundColor,
+  IntegrationConfig,
+  createIconComponentFromSimpleIconsSvgPath,
+  useIntegration,
+  IntegrationConfigOptions,
+} from '@techboi/consent-manager'
+
+import Matomo from 'simple-icons/icons/matomo'
+
+declare global {
+  interface Window {
+    _paq?: unknown[]
+  }
+}
+
+interface MatomoTrackerConfig extends IntegrationConfigOptions {
+  matomoURL?: string
+  siteID?: string
```

Comment:
When I make them optional, the error is gone.

https://github.com/techboi/consent-manager/pull/18#issuecomment-797136526


But these values are mandatory, otherwise matomo won't work. Ideas? ^^

Reply:
Ideally we need to make IntegrationConfig a generic to be able to accomodate for different required options. But that means we need to be able to distinguish between the different passed integrations.

For the moment to get us started: keep as is, we can tackle that for a later release.